### DISCUSSION TITLE
[V4] - How to get v4 going with Deno and a separate CSS entrypoint?

### DISCUSSION THREAD

**Discussion by bebraw** [tailwindlabs/tailwindcss#15339] [Help]
I set up a small demonstration at https://github.com/bebraw/tailwind4-demo . The problem is that the setup doesn't find `tailwindcss` for some reason when running through Deno.

It would be great if you could point me to a working example where an external CSS file is used as an entry point for loading Tailwind or help me amend this example to a working state.

**Comment by Hortasha** [comment]
I think I'm hitting onto the same issue. Tailwind works fine but the extension seem to be using v3.4.4 looking at the output.

Edit:
Apperently, in deno: 
Making sure that you have nodeModules in deno will help using tailwind 4.
"nodeModulesDir": "auto" for example.

**Comment by CarcajadaArtificial** [comment]
Has anyone found a consistent setup for a solution? When trying to run my tailwind configuration, I'm getting this error:

error: Uncaught (in promise) Error: It looks like you're trying to use `tailwindcss` directly as a PostCSS plugin. The PostCSS plugin has moved to a separate package, so to continue using Tailwind CSS with PostCSS you'll need to install `@tailwindcss/postcss` and update your PostCSS configuration.

**Comment by BrunoBernardino** [comment]
I also can't use `tailwindcss` v4 with `npx` in a non-node/deno/bun (js runtime) project (so there's no `package.json` nor `deno.json`), like `npx @tailwindcss/cli@4.1.5 -i ./css/input.css -o ./css/tailwind.css` will fail with the same error mentioned above, that it can't find `tailwindcss`, but `npx tailwindcss@3.4.17 -i ./css/input.css -o ./css/tailwind.css` works without a problem.
